jobs/release-monitoring/discovery/github/Jenkinsfile

```diff
@@ -240,6 +248,39 @@ node {
         isGARelease = hasNewGARelease(componentRelease, latestRelease, productName)
         println "[INFO] latestRelease:${latestRelease}, componentRelease:${componentRelease}, isGARelease:${isGARelease}"
         currentBuild.description = "latest: ${latestRelease}\ncurrent: ${componentRelease}"
+
+        if (releaseTagVar == "") {
+          productVersion = latestRelease
+        }
+
+        println "[INFO] product version: ${productVersion}"
+    }
+
+    stage('Fetch Component Product Version') {
+        when(productVersion == "") {
+          println "[INFO] Retrieving component product version"
+          def templateUrl = "https://raw.githubusercontent.com/${projectOrg}/${projectRepo}/${latestRelease}/${productVersionLocation}"
+
+          if (projectOrg == "integr8ly") {
+            productVersion = sh(returnStdout: true, script: "curl -i '${templateUrl}' | grep '${productVersionIdentifier}' | head -n 1 | awk '{print \$3}'")
+            productVersion = productVersion.replaceAll("[\"\']", "")
+          }
+
+          // Current tags/releases does not have the right product version in the webapp handler.
+          // This can be removed once this property has been updated with the correct information in the next release
+          // https://github.com/integr8ly/tutorial-web-app-operator/blob/v0.0.4/pkg/handlers/webhandler.go#L23
+          if (productName == "webapp") {
+            productVersion = sh(returnStdout: true, script: "curl -i '${templateUrl}' | grep '${productVersionIdentifier}' | awk -F ':' '{print \$3}'")
+          }
+
+          productVersion = productVersion.trim()
```

Comment:
Product version for webapp is returning v<major>.<minor>.<patch>, where this variable previously only had a semver version string. Maybe we should remove any additional chars like the v.